ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/forms/DeliveryDestinationsForm.tsx

```diff
@@ -12,21 +12,30 @@ import {
     Form,
     PageSection,
     Title,
+    Tooltip,
 } from '@patternfly/react-core';
-import { PlusCircleIcon, TrashIcon } from '@patternfly/react-icons';
+import { HelpIcon, PencilAltIcon, PlusCircleIcon, TrashIcon } from '@patternfly/react-icons';
 import { FormikProps } from 'formik';
+import resolvePath from 'object-resolve-path';
```

Comment:
About half the time we need this, we use lodash's `get`, and the other half we use this. Is there any difference in those two functions?

Reply:
That's a good point. I didn't really register that. We should keep using `get`, and try to remove this in the future. I'll make a note of that